weblate/templates/component-progress.html

```diff
@@ -28,14 +28,14 @@
 <div class="panel-footer">
 <form class="form-inline">
   <div class="form-group">
-<a href="{% url return_url path=object.get_url_path %}" class="btn btn-primary" id="progress-return">{% trans "Return to the component" %}</a>
+<a href="{% url return_url path=return_target.get_url_path %}" class="btn btn-primary" id="progress-return">{% trans "Return to the target" %}</a>
```

Comment:
```suggestion
<a href="{% url return_url path=return_target.get_url_path %}" class="btn btn-primary" id="progress-return">{% trans "Go back" %}</a>
```

Reply:
Just after I posted, I realized that this is not really “back” in some situations. Especially when a new component is being imported, you don't go back, but to the newly created component.

Reply:
```suggestion
<a href="{% url return_url path=return_target.get_url_path %}" class="btn btn-primary" id="progress-return">{% blocktranslate %}Continue to {{ return_target }}{% endblocktrans %}</a>
```